### DISCUSSION TITLE
Content Box position - full moves

### DISCUSSION THREAD

**Discussion by Cix75** [trufanov-nok/scantailor-universal#103] [Ideas]
Hello!
I just updated this nice, fantastic version of scantailor and I must say THANKS to the developers. It really help me each time.
I have two questions and I will try to explain the first one in this discussion.

Ones selected the content of a book/magazine and applied it on all pages (with the options menu) I will find the same dimension of the content box on all the pages as expected.

But, happens then that, for example, on a total of 200 pages it will not fit perfectly the content I really need on all pages (check the picture)
![2022-01-18_134017](https://user-images.githubusercontent.com/63376261/149940245-676f2fac-1f13-4cd2-bc6c-386f1363f12e.png)

so,I will need to adjust the border to set the right position.
The "Auto" option is not always precise and it also happens that some pixels of the characters or images are cut out, so I always do everything manually.

In another old version of scantailor there was a possibility of pressing the "SHIFT" key and, clicking on the content box, be able to move the whole box without changing its size and center it 'as is' on the necessary content.

There is this possibility here too maybe? 
if not, it could be really useful to have in one of the next releases, please.

Many thanks for what you can do and for your effort.

**Comment by trufanov-nok** [comment]
Hi,

I'm I right that you need a rectangle dragging? Like it's done for the image zones in Mixed mode at output processing step? Pressing and holding a left mouse button inside a zone while holding a SHIFT key will allow to move the zone across the page without changing its size.  
If it's so then it won't be difficult to implement for "Select content" rectangle although technically it's not a zone at all.

**Reply by Cix75** [reply]
> Hi,
> 
> I'm I right that you need a rectangle dragging? Like it's done for the image zones in Mixed mode at output processing step? Pressing and holding a left mouse button inside a zone while holding a SHIFT key will allow to move the zone across the page without changing its size. If it's so then it won't be difficult to implement for "Select content" rectangle although technically it's not a zone at all.

Hello, because I haven't gotten any good solution on this, I thought that maybe my request was not so clear to understand.
then, I recorded e video using an old version of ScanTailor advanced that can do it, to explain better what I can't be able to do on the v2.13 and I would really like to have this useful feature
https://www.dropbox.com/s/xn2xy7cg6l5z982/Video_2022-01-21_193514.wmv?dl=0

Hope you will be able to watch it.

regards.

**Reply by trufanov-nok** [reply]
Hi, I've got the idea - thanks. Will try to implement tomorrow. Are you a Win or Linux user? Bcs I'm more struggling with a Windows version builds then feature implementation

**Reply by Cix75** [reply]
Ok, thanks for your help and effort.
I am a Windows 10 user, take the time you need. The important thing for me was to be sure that I explained myself well so that you would understand the problem and it was not me who was missing something.
Hope to see it in a future release at least ;) crossed finger.

**Comment by Cix75** [comment]
thanks for the reply but it is not exactly what I mean.
I need just to move the selection as is, only the content box selection (the square blue).
If I push the SHIFT key + left button I will be able to move the whole page including the content box instead.

**Comment by trufanov-nok** [comment]
I have pushed a commit that adapts the Scan Tailor Advanced approach of moving the content selection area.  
In fact, I've realized that STU had this feature already. But to move the area you need to hover the mouse over the edge or corner and press SHIFT (by default). It didn't react on any action *inside* the area - borders only. That was a known issue, I suppose, but I forget that. Also the original approach behaved differently when moved zone is going out of page borders - it shrinks the zone. STA approach is more convenient, so I copied its piece of code and adapted it for STU. As a Win user you won't get this feature till one rebuild the Win release. But at least you may use the old feature (moving by borders) in current Win release.

**Reply by Cix75** [reply]
Ahhhh ok, thanks anyway.
Damn, I had tried all the possibilities but didn't think to press shift over the edge or corner.
anyway thanks for the help to understand :)
